CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## unreleased
 * Require Xcode 15.0+ and Swift 5.9+ (per [App Store requirements](https://developer.apple.com/news/?id=khzvxn8a))
 * Updated expiring pinned vendor SSL certificates
+* BraintreeDataCollector
+  * Bump to PPRiskMagnes v5.5.0-static-Xcode15-MinOSVersion100 with fix for Xcode 15.3 SPM validation bug
```

Comment:
```suggestion
  * Bump to PPRiskMagnes version 5.5.0 with fix for Xcode 15.3 Swift Package Manager validation bug (fixes #1229)
```

Reply:
Yes I think it's just missing in the diff.  I see the Privacy Manifest file.